From ff80107e1cca82430eceb5719bbb186e4d9ac480 Mon Sep 17 00:00:00 2001
From: "M. Vefa Bicakci" <vefa.bicakci@windriver.com>
Date: Fri, 15 Apr 2022 04:29:31 +0000
Subject: [PATCH 10/10] bnxt_en/Makefile: Allow parallelized build

This commit allows bnxt_en's build to be parallelized via the use of
"$(MAKE)" instead of "make". Without this change, make for bnxt_en
reports the following error and continues the build in a serialized
manner:

  warning: jobserver unavailable: using -j1. \
    Add '+' to parent make rule.

Signed-off-by: M. Vefa Bicakci <vefa.bicakci@windriver.com>
---
 bnxt_en-1.10.2-227.0.130.0/Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/bnxt_en-1.10.2-227.0.130.0/Makefile b/bnxt_en-1.10.2-227.0.130.0/Makefile
index e229cc9..84c16d2 100644
--- a/bnxt_en-1.10.2-227.0.130.0/Makefile
+++ b/bnxt_en-1.10.2-227.0.130.0/Makefile
@@ -1364,9 +1364,9 @@ endef
 
 default:
 ifeq ($(CROSS_COMPILE),)
-	make -C $(LINUX) M=$(shell pwd) $(call fwd_ver)modules
+	$(MAKE) -C $(LINUX) M=$(shell pwd) $(call fwd_ver)modules
 else ifneq ($(CROSS_COMPILE),)
-	make -C $(LINUXSRC) M=$(shell pwd) $(call fwd_ver)modules CROSS_COMPILE=$(CROSS_COMPILE) ARCH=$(ARCH)
+	$(MAKE) -C $(LINUXSRC) M=$(shell pwd) $(call fwd_ver)modules CROSS_COMPILE=$(CROSS_COMPILE) ARCH=$(ARCH)
 endif
 
 yocto_all:
-- 
2.43.0

